lifecycleScripts/runTests.js

```diff
@@ -0,0 +1,41 @@
+var spawn = require("child_process").spawn;
+var Promise = require("nodegit-promise");
+var path = require("path");
+
+var determineTarget = require("./determineTarget");
+
+return determineTarget()
+  .then(function(targetInfo) {
+    var opts = {
+      cwd: ".",
+      maxBuffer: Number.MAX_VALUE,
+      env: process.env,
```

Comment:
You might want to copy `process.env` here or else when you set `opts.env.ATOM_SHELL_INTERNAL_RUN_AS_NODE` down below it will actually be setting `process.env.ATOM_SHELL_INTERNAL_RUN_AS_NODE` as well I believe.